Security components are missing the version in the BOM

The Security guide suggest it's enough to add  the following to a Quarkus project:

```
<dependency>
   <groupId>io.quarkus</groupId>
   <artifactId>quarkus-elytron-security</artifactId>
</dependency>
```
This results currently in :

```
[ERROR] 'dependencies.dependency.version' for io.quarkus:quarkus-elytron-security:jar is missing.
```

--- a/bom/pom.xml
+++ b/bom/pom.xml
@@ -237,6 +237,12 @@
             </dependency>
             <dependency>
                 <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-elytron-security</artifactId>
+                <version>${project.version}</version>
+                <scope>provided</scope>
+            </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
                 <artifactId>quarkus-hibernate-validator</artifactId>
                 <version>${project.version}</version>
                 <scope>provided</scope>

